src/devices_models/devices/common/objective_function/common.jl

```diff
@@ -85,16 +85,18 @@ function add_shut_down_cost!(
     for d in devices
         PSY.get_must_run(d) && continue
 
+        add_as_time_variant = is_time_variant(PSY.get_shut_down(PSY.get_operation_cost(d)))
         for t in get_time_steps(container)
             my_cost_term = get_shutdown_cost_value(
                 container,
                 d,
                 t,
-                is_time_variant(PSY.get_shut_down(PSY.get_operation_cost(d))),
+                add_as_time_variant,
             )
             iszero(my_cost_term) && continue
-            exp = _add_proportional_term_variant!(
-                container, U(), d, my_cost_term * multiplier, t)
+            exp = _add_proportional_term_maybe_variant!(
+                Val(add_as_time_variant), container, U(), d, my_cost_term * multiplier,
+                t)
             # add_to_expression!(container, ProductionCostExpression, exp, component, t)  # TODO do we want this?
```

Comment:
Do we want to fix this too?

Reply:
My understanding is that on this first pass we aren't too concerned about `ProductionCostExpression`?